packages/components/sort-select/stories/index.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+import { useArgs } from '@storybook/preview-api';
```

Comment:
Should probably listen to the eslint bot here or it might turn the AI killer robots on you

Reply:
Whew thanks for the save, fixed! It was working and I assumed it imported from the `storybook` we already have installed.